codespell_lib/data/dictionary.txt

```diff
@@ -16281,6 +16296,7 @@ frequancies->frequencies
 frequancy->frequency
 frequant->frequent
 frequantly->frequently
+frequence->frequency
```

Comment:
While I suspect `frequence` is mostly used by French native speakers, this alternative spelling can be found in [Merriam-Webster](https://www.merriam-webster.com/dictionary/frequence), [Collins](https://www.collinsdictionary.com/dictionary/english/frequence), but not Cambridge or Oxford's Learners.

Perhaps in `dictionary_rare.txt`?

Reply:
Indeed. Moved to rare.